pkgs/development/python-modules/paddlepaddle/binary-hashes.nix

```
@@ -0,0 +1,31 @@
+{ pythonVersion, system }:
+let
+  all = {
+    x86_64-linux = {
+      platform = "manylinux1_x86_64";
+      hashes = {
+        cp39 = "sha256-Yu/FWoMhYp+behAth/jH0FKlf2LJr8TyvL9MBwmuews=";
+        cp310 = "sha256-O7d/5LY2dEMf5gW5WrN3xzIIEi2vT0RWoMeVOk5lATk=";
+      };
+    };
+    x86_64-darwin = {
+      platform = "macosx_10_9_x86_64";
+      hashes = {
+        cp39 = "sha256-5g9b2gC6uosMpoJiobpj8yToIS6ifAFRvLEqnc/o/QQ=";
+        cp310 = "sha256-2c1hjwNCOOOx9tVfBk+Pyk/pF0m/2tAmRsBH91834eM=";
+      };
+    };
+    aarch64-darwin = {
+      platform = "macosx_11_0_arm64";
+      hashes = {
+        cp39 = "sha256-JhYNTOx1UkuNf/63lHXBDry6FQjPnbIB8jU5jKcyX2k=";
+        cp310 = "sha256-4ltYEYm2OzPBc6D2bQt2dEh6Sz+5m1mMKGGYgQGLSAY=";
+      };
+    };
```

```suggestion
    };
    aarch64-linux = {
        ....
    };
```
just adding this here so we don't forget.
I think the ... are not so obscure, but let me know if you need more detailed advice.
(The reason is that ofborg will not be happy without all the main platforms being addressed or or explicitely disabled).

---

See https://github.com/NixOS/nixpkgs/pull/249297#issuecomment-1680769951
Maybe I can just add:

```nix
{
    aarch64-linux = {
      platform = "";
      hashes = {cp39 = ""; cp310 = "";};
    };
}
```

---

Just saw your comment about there not being a aarch64-linux distribution.
We should just disable it in the supported platforms then. Thanks for checking it out!

---

But isn't it already disabled via:

```nix
{platforms = [ "x86_64-linux" "x86_64-darwin" "aarch64-darwin" ];}
```

---

that is the fix I was about to suggest yeah.
let's leave the empty attrset
```
    aarch64-linux = {
      platform = "";
      hashes = {cp39 = ""; cp310 = "";};
    };
```
with a comment on top saying that this is just for ofborg.

---

Done